Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Rojo Yeah, for merging perhaps. From my side it shows conflits for `rebase and merge`. If we just merge it and deleted the branch, the the master could lose some ancesters in history. No big deal, but still cleaner history is better.